src/lib/core/proc_name_htab.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#pragma once
+
+#include "trivia/config.h"
+
+#ifdef ENABLE_BACKTRACE
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+#include "libunwind.h"
+
+/* Find procedure name and offset in hash table based on RIP register value. */
+char *
+proc_name_htab_find(unw_word_t ip, unw_word_t *offs);
```

Comment:
Better call it 'cache', not 'htab', because it's a cache, and a cache isn't necessarily implemented using a hash table.

Reply:
Yeah, I noticed that most of the hash tables we have are called _cache_s: the thing that boggled me is that they don't really behave as caches, i.e., they have no eviction.

But I now see that you are right in terms of abstractions.

Fixed: *_htab_* -> *_cache_*.